notebooks/run_model_in_notebook.ipynb

```diff
@@ -46,7 +46,7 @@
     "# path to config file\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        config_run = PyStemmusScope.read_config(config_path)
</pre><p>Replace it with <code>iostreamer.read_config</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>